### ISSUE TITLE
[Linux] Game cannot open with bumblebee

### ISSUE THREAD

**Issue by dracorl** [ValveSoftware/Dota-2#620]
Games working stable normally but trying to open with bumblebee game cant open and there is no error and log file. 
I am tried to verify game cache and use this commands "primusrun %command% -sw -con_logfile" and didt work.

```
Processor Information:
    Vendor:  GenuineIntel
    CPU Family:  0x6
    CPU Model:  0x2a
    CPU Stepping:  0x7
    CPU Type:  0x0
    Speed:  2900 Mhz
    4 logical processors
    2 physical processors
    HyperThreading:  Supported
    FCMOV:  Supported
    SSE2:  Supported
    SSE3:  Supported
    SSSE3:  Supported
    SSE4a:  Unsupported
    SSE41:  Supported
    SSE42:  Supported

Network Information:
    Network Speed:  

Operating System Version:
    Ubuntu 14.04.3 LTS (64 bit)
    Kernel Name:  Linux
    Kernel Version:  3.19.0-31-generic
    X Server Vendor:  The X.Org Foundation
    X Server Release:  11701000
    X Window Manager:  <Unknown>
    Steam Runtime Version:  steam-runtime-release_2015-06-12

Video Card:
    Driver:  Intel Open Source Technology Center Mesa DRI Intel(R) Sandybridge Mobile x86/MMX/SSE2

    Driver Version:  3.0 Mesa 10.5.9
    OpenGL Version: 3.0
    Desktop Color Depth: 24 bits per pixel
    Monitor Refresh Rate: 60 Hz
    VendorID:  0x10de
    DeviceID:  0xdf4
    Number of Monitors:  1
    Number of Logical Video Cards:  2
    Primary Display Resolution:  1366 x 768
    Desktop Resolution: 1366 x 768
    Primary Display Size: 13,54" x 7,60"  (15,51" diag)
                                            34,4cm x 19,3cm  (39,4cm diag)
    Primary VRAM Not Detected

Sound card:
    Audio device: Intel CougarPoint HDMI

Memory:
    RAM:  3862 Mb

Miscellaneous:
    UI Language:  English
    LANG:  en_US.UTF-8
    Microphone:  Not set
    Total Hard Disk Space Available:  596825 Mb
    Largest Free Hard Disk Block:  535950 Mb

Installed software:

Recent Failure Reports:
```

Can you help me?

**Comment by gdrewb-valve** [comment]
As this doesn't appear to be a Reborn issue I'm not sure there's much we can do but it's possible there's some bumblebee knowledge floating around.

**Comment by dracorl** [comment]
Do you know anything about that log?
Its from my terminal:

```
rm: cannot remove ‘/home/engin/.steam/bin’: Is a directory
Running Steam on ubuntu 14.04 64-bit
STEAM_RUNTIME has been set by the user to: /home/engin/.steam/ubuntu12_32/steam-runtime
ExecCommandLine: "/home/engin/.steam/root/ubuntu12_32/steam steam://open/driverhelperready"
ExecSteamURL: "steam://open/driverhelperready"
Generating new string page texture 235: 256x256, total string texture memory is 393,22 KB
Generating new string page texture 236: 256x256, total string texture memory is 2,93 MB
Game update: AppID 570 "Dota 2", ProcID 25467, IP 0.0.0.0:0
ERROR: ld.so: object '/home/engin/.steam/ubuntu12_32/gameoverlayrenderer.so' from LD_PRELOAD cannot be preloaded (wrong ELF class: ELFCLASS32): ignored.
ERROR: ld.so: object '/home/engin/.steam/ubuntu12_32/gameoverlayrenderer.so' from LD_PRELOAD cannot be preloaded (wrong ELF class: ELFCLASS32): ignored.
ERROR: ld.so: object '/home/engin/.steam/ubuntu12_32/gameoverlayrenderer.so' from LD_PRELOAD cannot be preloaded (wrong ELF class: ELFCLASS32): ignored.
pid 25471 != 25469, skipping destruction (fork without exec?)
ERROR: ld.so: object '/home/engin/.steam/ubuntu12_32/gameoverlayrenderer.so' from LD_PRELOAD cannot be preloaded (wrong ELF class: ELFCLASS32): ignored.
pid 25470 != 25469, skipping destruction (fork without exec?)
ERROR: ld.so: object '/home/engin/.steam/ubuntu12_32/gameoverlayrenderer.so' from LD_PRELOAD cannot be preloaded (wrong ELF class: ELFCLASS32): ignored.

malloc: unknown:0: assertion botched
free: called with unallocated block argument
last command: (null)
Aborting...Aborted (core dumped)
Game removed: AppID 570 "Dota 2", ProcID 25467 
```

**Comment by gdrewb-valve** [comment]
Some piece of code did something wrong but the log doesn't identify if it was in Bumblebee, the drivers or Reborn.

**Comment by Tele42** [comment]
Ubuntu Trusty-fixes is shipping libdrm-intel1 2.4.60, which is known incompatible and fixed in 2.4.61. For full details see https://github.com/ValveSoftware/steam-for-linux/issues/3506. Also, there is a posted workaround in this issue report at https://github.com/ValveSoftware/steam-for-linux/issues/3506#issuecomment-102565272.

**Comment by dracorl** [comment]
It worked. Thank you for your help.
